<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xvideoViewClass</class>
 <widget class="QWidget" name="xvideoViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>xvideoView</string>
  </property>
  <widget class="QLabel" name="video1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>201</width>
     <height>181</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 255);</string>
   </property>
   <property name="text">
    <string>video</string>
   </property>
  </widget>
  <widget class="QLabel" name="video2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>20</y>
     <width>181</width>
     <height>171</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>video2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open1">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>240</y>
     <width>75</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QComboBox" name="pix1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>YUV420P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RGBA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ARGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BGRA</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="width1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="height1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>320</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99997</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="set_fps1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="set_fps2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>360</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="height2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>320</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99997</number>
   </property>
   <property name="value">
    <number>300</number>
   </property>
  </widget>
  <widget class="QComboBox" name="pix2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>240</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>YUV420P</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RGBA</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ARGB</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>BGRA</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="width2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>290</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999</number>
   </property>
   <property name="value">
    <number>400</number>
   </property>
  </widget>
  <widget class="QPushButton" name="open2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QLabel" name="fps1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 0, 25);</string>
   </property>
   <property name="text">
    <string>FPS:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fps2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 8, 24);</string>
   </property>
   <property name="text">
    <string>FPS:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="yuvrgbplayer.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>open1</sender>
   <signal>clicked()</signal>
   <receiver>xvideoViewClass</receiver>
   <slot>Open1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>193</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>191</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>open2</sender>
   <signal>clicked()</signal>
   <receiver>xvideoViewClass</receiver>
   <slot>Open2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>447</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>450</x>
     <y>454</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Open1()</slot>
  <slot>Open2()</slot>
 </slots>
</ui>
